feature-dapp-impl/src/main/java/io/novafoundation/nova/feature_dapp_impl/utils/tabs/models/CurrentTabState.kt

```diff
@@ -0,0 +1,13 @@
+package io.novafoundation.nova.feature_dapp_impl.utils.tabs.models
+
+sealed interface CurrentTabState {
+
+    object NotSelected : CurrentTabState
+
+    data class Selected(val tab: BrowserTab, val pageSession: PageSession) : CurrentTabState
+}
+
+fun CurrentTabState.stateId() = when (this) {
+    CurrentTabState.NotSelected -> "not_selected"
```

Comment:
I dont like the idea of using the abritrary constant as a filler for NotSelected state. Since we only use this function for `distictUntilChanged` I suggest to just return null here